LICENSE-ISC

```diff
@@ -0,0 +1,15 @@
+ISC License (ISC)
+Copyright (c) 2021, Jacob Hoffman-Andrews <jsha@letsencrypt.org>
```

Comment:
Do you need a contributor agreement indicating that contributors agree to assign copyright to you, or is that implied by this line and the corresponding one in LICENSE-MIT?

Reply:
Honestly, no idea :-)